buildSrc/src/main/groovy/com/uber/okbuck/core/dependency/VersionlessDependency.groovy

```diff
@@ -10,13 +10,18 @@ class VersionlessDependency {
     final String group
     final String name
 
+    String classifier
```

Comment:
Done. This change adds the additional arguments to the constructor and pushes off the type logic here as well.